lib/src/metta/runner/stdlib_minimal.rs

```diff
@@ -412,7 +410,7 @@ impl Grounded for CaseOp {
             Ok(results) if results.is_empty() =>
                 // TODO: MINIMAL in minimal MeTTa we should use Empty in both
                 // places here and in (case ...) calls in code
```

Comment:
Could be removed as you done it in this PR:
```suggestion
                // TODO: MINIMAL in minimal MeTTa we should use Empty in both
                // places here and in (case ...) calls in code
```

Reply:
@Necr0x0Der please commit it before merge, I cannot for some reason.

Reply:
I cannot either, because the suggestion consists in replacing the comment with the same comment (not removing the fragment). But I can do this locally